com.unity.render-pipelines.universal/Runtime/DeferredLights.cs

```diff
@@ -423,12 +419,6 @@ internal bool IsRuntimeSupportedThisFrame()
                 };
             }
             this.DepthAttachmentHandle = this.DepthAttachment;
-#if ENABLE_VR && ENABLE_XR_MODULE
-            // In XR SinglePassInstance mode, the RTs are texture-array and all slices must be bound.
-            if (renderingData.cameraData.xr.enabled)
-                for (int i = 0; i < this.GbufferAttachmentIdentifiers.Length; ++i)
-                    this.GbufferAttachmentIdentifiers[i] = new RenderTargetIdentifier(this.GbufferAttachmentIdentifiers[i], 0, CubemapFace.Unknown, -1);
-#endif
```

Comment:
being this XR do we need any XR QA or specific testing to make sure we don't break anything? Or are Yamato XR tests enough?

Reply:
maybe @sandy-carter-unity would know best, but from what i see - these identifiers are not used anymore, so didn't see point of keeping them. This was basically the one of two places were the gbuffer identifiers were used. I highly doubt we still need XR specific QA for this, i think yamato XR would catch it if this broke anything